com.anydesk.Anydesk.json

```diff
@@ -92,6 +109,10 @@
                 {
                     "type": "patch",
                     "path": "gtkglext-gcc8.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "x11-libs_gtkglext_files_gtkglext-1.2.0-no-pangox.patch"
```

Comment:
As above, `"paths": []`

Reply:
If there is a patch here to remove a dependency on pangox, is pangox-compat still needed above?